From: Jan Engelhardt <jengelh@inai.de>
Date: 2012-05-29 13:24:33.672703853 +0200

Replace outdated constructs and replace dist-lzma by dist-xz.

Makefile.am:2: error: support for lzma-compressed distribution archives has been
autoreconf: automake failed with exit status: 1

---
 Makefile.am  |    1 -
 configure.ac |    7 ++++---
 2 files changed, 4 insertions(+), 4 deletions(-)

Index: libxspf-1.2.0/Makefile.am
===================================================================
--- libxspf-1.2.0.orig/Makefile.am
+++ libxspf-1.2.0/Makefile.am
@@ -1,5 +1,4 @@
 ## Process this file with automake to produce Makefile.in
-AUTOMAKE_OPTIONS = 1.10.1 gnu dist-zip dist-bzip2 dist-lzma subdir-objects
 
 ACLOCAL_AMFLAGS = -I m4
 
@@ -41,7 +40,9 @@ endif
 
 libxspf_la_CFLAGS = @URIPARSER_CFLAGS@
 
-libxspf_la_LDFLAGS = @URIPARSER_LIBS@ -version-info 4:1:0
+libxspf_la_LDFLAGS = -no-undefined -version-info 4:1:0
+
+libxspf_la_LIBADD = $(URIPARSER_LIBS)
 
 libxspf_la_SOURCES = \
 	src/XspfChunkCallback.cpp \
Index: libxspf-1.2.0/configure.ac
===================================================================
--- libxspf-1.2.0.orig/configure.ac
+++ libxspf-1.2.0/configure.ac
@@ -1,13 +1,16 @@
 # Process this file with autoconf to produce a configure script.
 AC_PREREQ(2.61)
-AC_INIT(src/XspfReader.cpp)
+AC_INIT([libxspf], [1.2.0])
+AC_CONFIG_SRCDIR(src/XspfReader.cpp)
 AC_CONFIG_AUX_DIR([build-aux])
-AM_INIT_AUTOMAKE(libxspf, 1.2.0)
+AM_INIT_AUTOMAKE([1.10.1 gnu dist-zip dist-bzip2 dist-xz subdir-objects])
 
-AC_PROG_LIBTOOL
-AC_PROG_INSTALL
-AC_PROG_CC
+AC_PROG_CC_STDC
+AC_USE_SYSTEM_EXTENSIONS
+AC_SYS_LARGEFILE
 AC_PROG_CXX
+LT_INIT([disable-static pic-only])
+AC_PROG_INSTALL
 PKG_PROG_PKG_CONFIG
 
 AC_HEADER_STDC
